deployments/helm/hephaestus/templates/buildkit/networkpolicy.yaml

```diff
@@ -19,4 +20,9 @@ spec:
         - podSelector:
             matchLabels:
               {{- include "hephaestus.controller.labels.matchLabels" . | nindent 14 }}
+          {{- if ne (include "hephaestus.buildkit.namespace" .) .Release.Namespace }}
+          namespaceSelector:
+            matchLabels:
+              {{- toYaml (lookup "v1" "Namespace" "" .Release.Namespace).metadata.labels | nindent 14 }}
```

Comment:
🤯 

This concerns me a bit if labels change, though I don't know if I have any evidence of that.

Reply:
I could see a situation where ` (lookup "v1" "Namespace" "" .Release.Namespace).metadata.labels` returns something like:

```yaml
# these don't change
kubernetes.io/metadata.name: <ns-name>
dominodatalab.com/node-pool: <bleh>

# however these could
custom-customer-label: foo
silly-openshift-metadata: v1.3.2.5.23423423
...
```

Adding labels is a non-issue but if users intentionally or inadvertently change the values of existing labels, then this shit breaks. We may want to offer the ability to override the selector to focus on a single k/v pair, right?

Reply:
Right, we have static labels like `domino-platform`, but those custom labels were my concern. An override couldn't hurt.